### ISSUE TITLE
File menu absent from the GUI

### ISSUE THREAD

**Issue by NicoKiaru** [NicoKiaru/ABBA-Python#6]

**Comment by NicoKiaru** [comment]
Probably related:

```
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException
	at javax.swing.SwingUtilities.replaceUIActionMap(SwingUtilities.java:1706)
	at javax.swing.plaf.basic.BasicMenuBarUI.uninstallKeyboardActions(BasicMenuBarUI.java:157)
	at javax.swing.plaf.basic.BasicMenuBarUI.uninstallUI(BasicMenuBarUI.java:129)
	at javax.swing.JComponent.uninstallUIAndProperties(JComponent.java:681)
	at javax.swing.JComponent.setUI(JComponent.java:658)
	at javax.swing.JMenuBar.setUI(JMenuBar.java:144)
	at javax.swing.JMenuBar.updateUI(JMenuBar.java:153)
	at sc.fiji.bdvpg.scijava.BdvScijavaHelper.clearBdvHandleMenuBar(BdvScijavaHelper.java:67)
	at ch.epfl.biop.atlas.aligner.gui.bdv.BdvMultislicePositionerView.lambda$clearBdvDefaults$22(BdvMultislicePositionerView.java:236)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by NicoKiaru** [comment]
Fixed by increasing java abba version from 0.3.5 to 0.3.6
